bom/pom.xml

```diff
@@ -98,8 +98,8 @@
         <wildfly-elytron.version>2.0.0.Alpha4</wildfly-elytron.version>
         <jboss-modules.version>1.8.7.Final</jboss-modules.version>
         <jboss-threads.version>3.0.0.Beta4</jboss-threads.version>
-        <vertx.version>3.7.0</vertx.version>
-        <httpclient.version>4.5.7</httpclient.version>
+        <vertx.version>3.7.1</vertx.version>
```

Comment:
Don't without my other PR or things are going to be broken.

Reply:
CI tests passed, what could go wrong :-D ?